ci-operator/step-registry/openshift-qe/installer/bm/prereqs/openshift-qe-installer-bm-prereqs-commands.sh

```
@@ -19,17 +19,33 @@ export QUADS_INSTANCE
 LOGIN=$(cat "${CLUSTER_PROFILE_DIR}/login")
 export LOGIN
 
+if [[ "${OCP_INVENTORY_OVERRIDE}" == "true" ]]; then
+  OCPINV=$(cat ${CLUSTER_PROFILE_DIR}/ocp_inventory_path)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)